packages/collector/test/tracing/frameworks/express/app.mjs

```diff
@@ -13,6 +13,13 @@ process.on('SIGTERM', () => {
   process.exit(0);
 });
 
+import mock from 'mock-require';
+const EXPRESS_VERSION = process.env.EXPRESS_VERSION;
+const EXPRESS_REQUIRE = EXPRESS_VERSION === 'latest' ? 'express' : `express-${EXPRESS_VERSION}`;
+if (EXPRESS_REQUIRE !== 'express') {
+  mock('express', EXPRESS_REQUIRE);
+}
+
 import instanaFactory from '../../../../src/index.js';
```

Comment:
issue: The express esm app should not import our collector here, it should use `--import` or `--experimental-loader`.

Reply:
Removed

Reply:
Ahh we do import the collector here because we use the SDK.

Reply:
https://github.com/instana/nodejs/pull/1261/files/99f3a8a60355974e2c12f62fd937bea7555f261b

Reply:
Added a comment in the test.